Sources/SwiftKuery/ConnectionPool.swift

```diff
@@ -228,7 +228,7 @@ public class ConnectionPool {
     // Items can therefore be borrowed or permanently removed with this method.
     //
     // This function will block until an item can be obtained from the pool. 
-    private func take() -> Connection? {
+    internal func take() -> Connection? {
```

Comment:
Revisit why this is now internal.

Reply:
This is because of the changes in connection pool connection relating to wrappers with a nil connection